src/components/containers/seo.js

```diff
@@ -24,7 +25,10 @@ const SEO = ({ description, meta, title, no_index }) => {
             }
         `,
     )
-
+    let no_index_staging
+    if (isBrowser() && window.location.hostname == 'dev.deriv.com') {
```

Comment:
you can either use `isStaging` from websocket config or `process.env.GATSBY_ENV === 'staging'`